lib/src/platform/src/realtime/realtime.dart

```diff
@@ -37,7 +37,7 @@ class Realtime extends PlatformObject {
     );
     _realtimeInstances[handle] = this;
 
-    if (io.Platform.isAndroid && options.autoConnect != false) {
+    if (io.Platform.isAndroid && !options.autoConnect) {
```

Comment:
it was `!= false` aka `true` now it's negation to `bool` which (in most cases) is `false` - is this intended?

Reply:
Thanks, fixed in 54fcf8e89d5e1504c9d9996bf4879f600b2b08a8